docs/site/migration/mounting-lb3app.md

```diff
@@ -25,7 +25,7 @@ Explorer.
 
 {% include note.html content="
 If you are not familiar with LoopBack 4, visit
-[Getting started](Getting-started.md) for an introduction.
+[Getting started](Getting-started) for an introduction.
```

Comment:
I believe the reason for a broken link is because the reference to Getting-started.md is not correct. Here is my suggested change:

```suggestion
[Getting started](../Getting-started.md) for an introduction.
```